image_client.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, config=None):
         ptc_timezone = timezone(timedelta(hours=-8), name="PST")
         self.datetime_now = datetime.now(ptc_timezone)
         self.update_time_delta()
-        if config is not None:
-            self.config = config
+        self.config = config
+        if config is not None and config.MAILING_LIST:
             self.monitoring_tools = MonitoringTools(config=config, report_path=config.REPORT_PATH)
```

Comment:
you have a conditional at the derived class level around the mailing list (botany importer and ich importer) and another one here - probably should have one or the other? Is there a case where we could get in trouble if we don't check both? Maybe some kind of cleverness around the config file where the image_client.py knows to check a vaiable named $COLLECTION_mailing_list (some variant on that) and we don't need the check in the derived class.